src/api-services/pagination/dto/Pagination.ts

```diff
@@ -0,0 +1,21 @@
+import { ApiProperty } from '@nestjs/swagger'
+
+export class Pagination {
+  @ApiProperty({
+    description: 'Current page returned by the call',
+    example: 1
+  })
+    pageNumber: number
+
+  @ApiProperty({
+    description: 'Amount of items in the page returned by the call',
+    example: 10
+  })
+    pageSize: number
+
+  @ApiProperty({
+    description: 'Total number of items in the list',
```

Comment:
Je comprend pas trop le `the list` 🤔

Reply:
C'est superflu: c'est le nombre total d'items..... À supprimer, bien vu.